runhouse/resources/hardware/cluster.py

```diff
@@ -116,6 +116,9 @@ def __init__(
         if self._default_env and not self._default_env.name:
             self._default_env.name = _unnamed_default_env_name(self.name)
 
+        self._tunnel_timeout = kwargs.get("tunnel_timeout")
```

Comment:
should this be `self._tunnel_timeout = kwargs.get("tunnel_timeout", TUNNEL_TIMEOUT)` and similarly with num_ports_to_try below?

Reply:
deferring to donny's temp approach for now , we should revisit this when we make it a real setting